python/ray/experimental/test/async_test.py

```diff
@@ -1,20 +1,18 @@
 import asyncio
```

Comment:
can we move this test to python/ray/tests/test_async?

Reply:
Sure!